.github/workflows/kind-integration.yml

```diff
@@ -114,20 +115,12 @@ jobs:
         run: |
           kubectl create namespace ${{ env.DSPA_EXTERNAL_NAMESPACE }}
 
-      - name: Create Minio Secrets
+      - name: Apply MariaDB and Minio Secrets and Configmaps in the External Namespace
+        working-directory: ${{ github.workspace }}/.github/resources/external-pre-reqs
         run: |
-          kubectl create secret generic minio --from-literal=accesskey=accesskey --from-literal=secretkey=secretkey -n ${{ env.DSPA_EXTERNAL_NAMESPACE }}
-
-      - name: Create MariaDB Secrets
-        run: |
-          password=$(kubectl get secret ds-pipeline-db-test -n test-mariadb -o jsonpath="{.data.password}" | base64 --decode)
-          kubectl create secret generic ds-pipeline-db-test --from-literal=password="$password" -n ${{ env.DSPA_EXTERNAL_NAMESPACE }}
-
-      - name: Apply rootCA ConfigMap
-        env:
-          RESOURCES_DIR: ${{ github.workspace }}/.github/resources
-        run: |
-          kubectl apply -f ${{ env.RESOURCES_DIR }}/tls/root-ca-configmap.yaml -n ${{ env.DSPA_EXTERNAL_NAMESPACE }}
+          cp ../minio/secret.yaml ./minio-secret.yaml
+          cp ../mariadb/secret.yaml ./mariadb-secret.yaml
```

Comment:
this is making it to dependent on the workflow, it also makes the `kustomization.yaml` file invalid in the `external-pre-reqs` folder, can you copy/paste these files into the folder in the git repo?

Reply:
The thought behind adding these statements was to avoid duplicate config files in code, but yes it makes sense to add them back so we could run kustomize locally too and not depend on the workflow.

Reply:
@DharmitD I think you can refer to these files in `kustomization.yaml` as   `- ../mariadb/secret.yaml` to avoid duplicating them.

Reply:
I'd tried it but doesn't work:

```
Error: accumulating resources: accumulation err='accumulating resources from '../mariadb/secret.yaml': security; file '/home/runner/work/data-science-pipelines-operator/data-science-pipelines-operator/.github/resources/mariadb/secret.yaml' is not in or below '/home/runner/work/data-science-pipelines-operator/data-science-pipelines-operator/.github/resources/external-pre-reqs'': must build at directory: '/home/runner/work/data-science-pipelines-operator/data-science-pipelines-operator/.github/resources/mariadb/secret.yaml': file is not directory
```

the file needs to be in the same directory for kustomize to work.